addons/l10n_si/__openerp__.py

```diff
@@ -20,23 +20,24 @@
 ##############################################################################
 
 {
-    "name" : "Slovenian - Accounting",
-    "version" : "1.2",
-    "author" : "Mentis d.o.o.",
-    "website" : "http://www.mentis.si",
-    "category" : "Localization/Account Charts",
-    "description" : " ",
-    "depends" : ["account", "base_iban", "base_vat", "account_chart", "account_cancel"],
-    "description" : "Kontni načrt za gospodarske družbe",
-    "data" : [
-        "data/account.account.type.csv", 
-        "data/account.account.template.csv",
-        "data/account.tax.code.template.csv",
-        "data/account.chart.template.csv",
-        "data/account.tax.template.csv",
-        "data/account.fiscal.position.template.csv",
-        "data/account.fiscal.position.account.template.csv",
-        "data/account.fiscal.position.tax.template.csv",
+    "name": "Slovenian - Accounting",
+    "version": "1.0",
```

Comment:
Use numbering scheme 8.0.x.y.z. This version number should be greater than 1.2, that its the previous version.